config/database.yml

```diff
@@ -86,11 +86,9 @@ test:
 # You can use this database configuration with:
 #
 #   production:
-#     url: <%= ENV['DATABASE_URL'] %>
+#     url: <%= ENV.fetch("DATABASE_URL") %>
 #
 production:
   <<: *default
-  database: PracticalDeveloper_production
-  pool: <%= ENV['DATABASE_POOL_SIZE'] %>
-  username: PracticalDeveloper
-  password: <%= ENV['PRACTICALDEVELOPER_DATABASE_PASSWORD'] %>
+  url: <%= ENV.fetch("DATABASE_URL", "") %>
```

Comment:
same, `.fetch` is convenient if there's a valued default or if we want to raise the exception if the value is missing, but if we default to an empty string, then there's no need for `.fetch`